pages/resources/[page].js

```diff
@@ -56,10 +56,10 @@ function Resources() {
   ];
 
   const initialValues = {
-    category: [],
-    q: '',
-    languages: [],
-    paid: '',
+    category: category || '',
+    q: q || '',
+    languages: Array.isArray(languages) ? languages : [languages].filter(Boolean),
+    paid: paid || '',
```

Comment:
I hope you don't mind, but I got rid of the nullish coalescing here. It's totally valid, but I'm trying to keep less sugar out of the codebase. I should add a lint rule to prevent it and optional chaining.